File: onlineexamination/src/main/java/com/cg/onlineexamination/student/dto/TestPaperdto.java
package com.cg.onlineexamination.student.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TestPaperdto {

	private int testPaperId;
	private String difficultyLevel;
	private String description;
	private String course;
	private int noOfQuestions;

}

File: src/app/test-paper-dto.ts
export class TestPaperDto {

    testPaperId:number;
	difficultyLevel:string;
	description:string;
	course:string;
	noOfQuestions:string;

    constructor(

    testPaperId:number,
	difficultyLevel:string,
	description:string,
	course:string,
	noOfQuestions:string


    ){

        this.testPaperId = testPaperId;
        this.difficultyLevel = difficultyLevel;
        this.description = description;
        this.course = course;
        this.noOfQuestions = noOfQuestions;
    }

}
